modules/op-tkn-trigger-management.adoc

```diff
@@ -75,3 +96,33 @@ List TriggerTemplates.
 ----
 $ tkn triggertemplate list -n myspace
 ----
+== clustertriggerbinding
+Manage ClusterTriggerBindings.
+
+.Example: Display ClusterTriggerBindings help
+----
+$ tkn clustertriggerbinding -h
+----
+
+== clustertriggerbinding delete
+Delete a ClusterTriggerBinding.
+
+.Example: Delete `myclusterbinding1` and `myclusterbinding2` ClusterTriggerBindings
+----
+$ tkn clustertriggerbinding delete myclusterbinding1 myclusterbinding2
+----
+== clustertriggerbinding describe
+Describe a ClusterTriggerBinding.
+
+.Example: Describe the `myclusterbinding` ClusterTriggerBinding
+----
+$ tkn clustertriggerbinding describe mybinding
+----
+
+== clustertriggerbinding list
+List ClusterTriggerBindings.
+
+.Example: List all the ClusterTriggerBindings
```

Comment:
either remove "the" from the title or say "all of the"

Reply:
done